core/jazz_ui/e2e/03-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ *Copyright 2016-2017 T Mobile, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); You may not use
+ * this file except in compliance with the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser, element, by, protractor, $} from 'protractor';
+import { Login } from '../page-objects/login.po';
+import { Jazz } from '../page-objects/jazzservices.po';
+import { CONFIGURATIONS } from '../../src/config/configuration';
+import { Timeouts } from 'selenium-webdriver';
+import { SSL_OP_SSLEAY_080_CLIENT_DH_BUG } from 'constants';
+
+
+const timeOutHigh = 2500000;
+const emailId = CONFIGURATIONS.optional.general.e2e.EMAIL_ID;
+
+describe('Overview', () => {
+  let login_po: Login;
+  let jazzServices_po: Jazz;
+  const EC = protractor.ExpectedConditions;
+  let tagging_count;
+  let winhandle;
+  let servicename;
+
+  beforeAll(() => {
+    login_po = new Login();
+    jazzServices_po = new Jazz();
+    });
+
+
+
+      function createservice(servicename)
+      {
+        //servicename='servicename' + Date.now();
```

Comment:
Remove commented lines (in other files as well).

Reply:
Yes.

Reply:
Removed All except 2 lines that we need for jazz internal